src/models.rs

```diff
@@ -154,3 +215,35 @@ impl Paged<Cat> {
         }
     }
 }
+#[derive(Debug, Snafu)]
+pub enum Error {
+    #[snafu(display("Missing or none attribute"))]
+    MissingAttribute { backtrace: Backtrace },
+
+    #[snafu(display("Request failure"))]
+    CynicRequestError {
+        source: CynicReqwestError,
+        backtrace: Backtrace,
+    },
+
+    #[snafu(display("Request failure"))]
+    RequestError {
+        source: reqwest::Error,
+        backtrace: Backtrace,
+    },
+
+    #[snafu(display("{:?}", message))]
+    RequestResultedInError { message: String },
+
+    #[snafu(display("Environment variable missing"))]
+    EnvVarMissing {
+        source: VarError,
+        backtrace: Backtrace,
+    },
+
+    #[snafu(display("Invalid header value"))]
+    InvalidHeaderValue {
+        source: InvalidHeaderValue,
+        backtrace: Backtrace,
+    },
+}
```

Comment:
That's copy pasted from `lib.rs`. 
For now it's ok. I'll figure something out to reduce code dup.